Pin Dropper/PinDropViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  PinDropViewController.swift
+//  Pin Dropper
+//
+//  Created by Talha Babar on 7/1/15.
+//  Copyright (c) 2015 Talha Babar. All rights reserved.
+//
+
+import UIKit
+
+class PinDropViewController: UIViewController, GMSMapViewDelegate {
+
+    let segueIdentifier = "ToPinsList"
```

Comment:
`segueIdentifier` can be the identifier for any segue. e.g when you have to go to multiple VCs from this VC....

So, the name should be more descriptive, something like `let PinsListSegueIdentifier`. One point to note here is that primitive type constants should start with a capital letter. Let me know if this makes sense... Otherwise, let's talk about this